app/models/test_track/lazy_visitor_by_identity.rb

```diff
@@ -0,0 +1,38 @@
+class TestTrack::LazyVisitorByIdentity
+  def initialize(identity)
+    @identity = identity
+  end
+
+  def loaded?
+    @visitor.present?
+  end
+
+  def id_loaded?
+    loaded?
+  end
+
+  private
+
+  def method_missing(method, *args, &block) # rubocop:disable Style/MethodMissing
+    __visitor__.send(method, *args, &block)
+  end
```

Comment:
Rubocop is wrong here too. If you want to delegate all the things, you don't want to do the method-whitelist-else-super dance they suggest. SimpleDelegator wouldn't be a thing if that advice were universal.